compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/QueryDesugar.java

```diff
@@ -925,6 +937,28 @@ BLangVariableReference addSelectFunction(BLangBlockStmt blockStmt, BLangSelectCl
         return getStreamFunctionVariableRef(blockStmt, QUERY_CREATE_SELECT_FUNCTION, Lists.of(lambda), pos);
     }
 
+    /**
+     * Desugar onConflictClause to below and return a reference to created onConflict _StreamFunction.
+     * _StreamFunction onConflictFunc = createOnConflictFunction
+     * @param blockStmt
+     * @param onConflictClause
+     * @param stmtsToBePropagated
+     * @return
+     */
```

Comment:
Improve or remove docs.

Reply:
updated in [commit](https://github.com/ballerina-platform/ballerina-lang/pull/41733/commits/e2c20f614268bbaea5d3752dd3ca5bb1e52bcaba)